instrumentation/restlet/restlet-1.0/library/src/main/java/io/opentelemetry/instrumentation/restlet/v1_0/RestletNetAttributesExtractor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.restlet.v1_0;
+
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetAttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.restlet.data.Request;
+import org.restlet.data.Response;
+
+final class RestletNetAttributesExtractor extends NetAttributesExtractor<Request, Response> {
+  @Override
+  public String transport(Request request) {
+    return SemanticAttributes.NetTransportValues.IP_TCP;
+  }
+
+  @Override
+  public @Nullable String peerName(Request request, @Nullable Response response) {
+    return request.getHostRef().getUserInfo();
```

Comment:
Is this the remote (client) host name?

Reply:
Looked into that and it wasn't the client host name. Seems like these versions of restlet allow only to retrieve the IP address, so I'm changing this to return null